File: android/app/src/main/java/com/example/bbokkibbokki/SplashActivity.java
package com.example.bbokkibbokki;

public class SplashActivity : AppCompatActivity() {



        override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)



        val intent = Intent(this, MainActivity::class.java)            // 실제 사용할 메인 액티비티

        startActivity(intent)

        finish()

        }

        }

File: app/src/main/java/com/example/bbokkibbokki/SplashActivity.kt
package com.example.bbokkibbokki

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
    private val SPLASH_TIME_OUT:Long = 1000 //

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        //main화면 불러오기
        Handler().postDelayed({
            startActivity(Intent(this, MainActivity::class.java))
            finish()
        }, SPLASH_TIME_OUT)
    }
}
